tests/testthat/_snaps/r.md

```diff
@@ -1,3 +1,21 @@
+# use_test_helper() creates a helper file
+
+    Code
+      use_test_helper(open = FALSE)
+    Condition
+      Error in `use_test_helper()`:
+      x Your package must use testthat to use a helper file.
+      Call `usethis::use_testthat()` to set up testthat.
```

Comment:
This should be a "todo" bullet, so I'm going to open a separate issue about that. Has nothing to do with this PR.